app/models/changelog.php

```diff
@@ -114,3 +108,27 @@
 
     return $link;
 };
+
+// Helper to generate change author names
```

Comment:
Not sure what you wanted to tell here. Maybe a link to the GitHub account makes more sense? If there's code, there's going to be a GitHub account.